settings.py

```diff
@@ -2015,6 +2020,9 @@ def _pkce_required(client_id):
 
 REST_FRAMEWORK = {
     'DATETIME_FORMAT': '%Y-%m-%dT%H:%M:%S.%fZ',
+    'DEFAULT_AUTHENTICATION_CLASSES': (
+       'rest_framework.authentication.TokenAuthentication',
+   ),
```

Comment:
is this needed? it seems like you are specifying auth classes for all of your views already

Reply:
Custom auth class is inheriting from `TokenAuthentication` and overrides `authenticate` method. This is then called by the framework as part of decorator `@authentication_classes((ABDMUserAuthentication,))` in the views.

Reply:
Yes, I see that, but if you are specifying the auth class on all the views then there is no need to set a default for the whole of CommCare.